__tests__/unit/core-http-utils/create-secure-server.test.ts

```diff
@@ -0,0 +1,19 @@
+import { createSecureServer } from "../../../packages/core-http-utils/src/server/create-secure";
+
+describe("createSecureServer ", () => {
+    it("callback should be executed", async () => {
+        const theCallback = jest.fn();
+        const smallFilePath = "./node_modules/http-proxy-agent/test/ssl-cert-snakeoil";
+        createSecureServer(
+            { host: "0.0.0.0", port: 3000 },
+            theCallback,
+            { key: smallFilePath + ".key", cert: smallFilePath + ".pem", port: "4007" },
+            [require("hapi-rate-limit")],
+        );
+
+        // This tests functionality from createServer, which theoretically create duplicative error messages.
+        // The right way to resolve this, IMO, is to make a plugin which makes the server secure, rather than having createSecureServer()
+        theCallback();
```

Comment:
I'm not really sure what to test here. 

I could mock createServer. But as noted in the comment, seems more reasonable to refactor away the method createSecureServer, and just use a plugin.

Reply:
https://github.com/ArkEcosystem/core/pull/2730